dev-docs/modules/audigentRtdProvider.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: page_v2
+title: Audigent Segmentation Real-time Data Module
+description: Audigent Segmentation RTD Module
+display_name: Audigent Segmentation
+module_code : audigent
+module_type : rtd
+display_name : Audigent
+enable_download : true
+sidebarType : 1
+---
+
+# Audigent Segmentation RTD Module
+{:.no_toc}
+
+* TOC
+{:toc}
+
+Audigent is a next-generation data management platform and a first-of-a-kind
+"data agency" containing some of the most exclusive content-consuming audiences
+across desktop, mobile and social platforms.
+
+This real-time data module provides quality user segmentation that can be
+attached to bid request objects destined for different SSPs in order to optimize
+targeting and increase publisher revenue. Audigent maintains a large database
+of first-party Tradedesk Unified ID, Audigent Halo ID and other id provider
+mappings to various third-party segment types that are utilizable across
+different SSPs. With this module, these segments can be retrieved and supplied
+to the SSP in real-time during the bid request cycle.
+
+## Publisher Usage
+
+Compile the audigent RTD module into your Prebid build:
+
+`gulp build --modules=userId,unifiedIdSystem,rtdModule,audigentRtdProvider,appnexusBidAdapter`
```

Comment:
technically you don't need to specify either `userId` or `rtdModule` here -- they should be included by the sub-modules that require them.